pkg/operator/sync.go

```diff
@@ -14,6 +36,16 @@ func syncControllers(c serviceCAOperator, operatorConfig *operatorv1.ServiceCA)
 		return err
 	}
 
+	// Remove resources related to the 4.4 controller deployment at most once. These
```

Comment:
@deads2k Is it acceptable to remove the resources related to the deployment at most once rather than continually, since the resources in question have no operational impact without the deployment (which is continually removed by the starter)?

Reply:
I don't care too much. It's good enough to do it once.